### PR TITLE
Allow for custom base URIs for GDPR support

### LINKED ISSUE #187: Support additional API hosts
I've built a Zoom integration for a client in the EU, which is of course subject to the GDPR.  One of their requests is that our integration use the GDPR compliant endpoint at https://eu01api-www4local.zoom.us/[1].  The current code does not appear to have a way to override the API host.  It would be nice if we could override that.

1: https://marketplace.zoom.us/docs/api-reference/introduction

### PR DESCRIPTION
**Pull request by prschmid**
Closes #187

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -5,7 +5,7 @@ isolated_build = true
 
 [testenv]
 deps =
-    -requirements-tests.txt
+    -rrequirements-tests.txt
```
Comment (prschmid) [inline tox.ini:8]: @mfldavidson I know you removed the extra `r` in https://github.com/prschmid/zoomus/commit/d16dc043d1d27aa638db262e75ead7f407af93a8, but looks like this isn't a typo and travis fails.  See this failing build (and then the subsequent fix). https://travis-ci.org/github/prschmid/zoomus/jobs/764225842

**Review by gregorydlogan — CHANGES_REQUESTED** [review]
Minor nits noticed in testing.

Might also be worth adding a note that the EU endpoint will not function unless your account is an EU account.  Neither my credentials, nor my clients work on the EU endpoints yet, but my client is due to migrate tomorrow.  Will try to remember to check again.

**Comment by prschmid** [comment]
@gregorydlogan Thanks for testing it out! I just pushed a commit with the added `/v2` to the `base_url`, and updated the documentation based on your suggestion.

Can you let me know how things pan out with your testing tomorrow with the EU enabled account? Once you give me the 👍 , I'll get this merged in. 

Thanks!

**Comment by prschmid** [comment]
@gregorydlogan Glad to hear that all went well! Congrats! Thanks for your help in testing and will merge this in now.

**Pull request merged**